resources/asset/js/morphext.min.js

```diff
@@ -0,0 +1,19 @@
+/*! Morphext - v2.4.5 - 2015-08-26 */ ! function(a) {
+    "use strict";
```

Comment:
sử dụng nháy đơn

Reply:
e sửa rồi ạ, nhưng toàn file có sẵn trong template thôi mà c